.github/workflows/backend.yml

```diff
@@ -3,31 +3,77 @@ on:
   pull_request:
     types: ['opened', 'edited', 'reopened', 'synchronize']
     paths:
-    - 'kafka-ui-api/**'
+      - 'kafka-ui-api/**'
```

Comment:
we're also lack 'kafka-ui-e2e-checks/**', so this workflow won't be triggered , when we change tests only